service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredTaskExecutor.java

```diff
@@ -73,4 +73,6 @@ public void execute(GetGpcStructuredTaskDefinition structuredTaskDefinition) {
 
         detectTranslationCompleteService.beginSendingCompleteExtract(ehrExtractStatus);
     }
+
+
```

Comment:
Try to check your changelists before committing / pushing to avoid unnecessary changes